lib/cdo/script_constants.rb

```diff
@@ -0,0 +1,54 @@
+# Info on existing tutorial scripts on Code Studio,
+# sometimes referenced in Pegasus.
+# Used for conditional behaviors.
+
+module ScriptConstants
+  HOC_2013_NAME = 'Hour of Code' # this is the old (2013) hour of code
```

Comment:
I feel like there's a way to make this an enum, so we don't have to extend tests every time we write something new